src/pyproject_fmt/cli.py

```diff
@@ -88,6 +92,12 @@ def _build_cli() -> ArgumentParser:
         default=DEFAULT_INDENT,
         help="number of spaces to indent",
     )
+    parser.add_argument(
+        "--max-supported-python",
+        type=Version,
```

Comment:
Let me know if know if this should just instead be a string and we parse it later. The advantage of having it here is that the users sees nicer error messages instead of large tracebacks:

```console
$ pyproject-fmt --max-supported-python=abc pyproject.toml
usage: pyproject-fmt [-h] [-V] [-s | --check] [--keep-full-version] [--indent INDENT]
                     [--max-supported-python MAX_SUPPORTED_PYTHON]
                     inputs [inputs ...]
pyproject-fmt: error: argument --max-supported-python: invalid Version value: 'abc'

```